templates/ingress.yaml

```diff
@@ -16,28 +17,8 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
-    kubernetes.io/tls-acme: {{ eq .Values.ingress.acme true | quote }}
-    nginx.ingress.kubernetes.io/custom-http-errors: "403,404,502,503"
-  {{- if .Values.ingress.auth.enabled }}
-    nginx.ingress.kubernetes.io/auth-url: https://houston.{{ .Values.ingress.baseDomain }}/v1/authorization
-    nginx.ingress.kubernetes.io/auth-signin: https://app.{{ .Values.ingress.baseDomain }}/login
-    nginx.ingress.kubernetes.io/auth-response-headers: authorization, username, email
-    nginx.ingress.kubernetes.io/proxy-cookie-path: / /{{ .Release.Name }}/
-    # To support people who are using Airflow's experimental API on the old URL, but don't
-    # have their client set up to follow redirects correctly: We proxy it to the right location
-    # for them.
-    nginx.ingress.kubernetes.io/server-snippet: |
-      location ^~ /api/ {
-        if ($host = '{{- include "deployments_subdomain" . }}' ) {
-          return 404;
-        }
-        proxy_pass {{ printf "https://%s/%s/airflow/$request_uri" ( include "deployments_subdomain" .) .Release.Name }};
-      }
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      if ($host = '{{- include "airflow_subdomain" . }}' ) {
-        return 308 {{ printf "https://%s/%s/airflow/$request_uri" ( include "deployments_subdomain" .) .Release.Name }};
-      }
+  {{- range $k, $v := .Values.ingress.webserverAnnotations }}
+    {{ $k | quote }}: {{ tpl $v $ctx | toYaml }}
```

Comment:
Please rework this logic to be simpler like just 

{{- .Values.ingress.webserverAnnotations }}

with maybe a wrapping conditional

Reply:
it is because some of the string include templating?
```
nginx.ingress.kubernetes.io/auth-signin: https://app.{{ .Values.ingress.baseDomain }}/login
```

Reply:
ok I changed my mind.